FreeRTOS/FreeRTOS-Kernel
# Replace <pre> with @code - remaining files

Description
-----------
This PR finalizes replacement of \<pre></pre> with doxygen's @code{c} @endcode in remaining files.

Test Steps
-----------
Check generated documentation

Related Issue
-----------
no issue

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
